Sources/ProjectSpec/Project.swift

```diff
@@ -132,9 +132,18 @@ extension Project: Equatable {
 
 extension Project {
 
+    @available(*, deprecated, message: "Use `Project.Spec` and `init(spec:basePath:)`.")
```

Comment:
Do these functions need to be deprecated? They will probably still be the main entry point. Consumers shouldn't have to care about `Spec`, that's just an implementation detail (Can even be an `internal` struct)

Reply:
The reason I deprecated these functions is that they will continue to function in the legacy behaviour, i.e. they won't respect the relative paths of included specs. I initially erased them completely but didn't want to break the public API so put them back with deprecation. Your call though obviously.